testcase/dynamicprivs/go.mod

```
@@ -0,0 +1,48 @@
+module github.com/pingcap/tipocket/testcase/dynamicprivs
+
+go 1.15
```

```suggestion
go 1.16
```
Please bump the version to 1.16